### PR TITLE
Forms Automated Test - LPS-131202 | master

### PR DESCRIPTION
**Pull request by PauloVAS**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsEntries.testcase [hunk 1]
```diff
@@ -143,13 +143,54 @@ definition {
...
+		Form.publishForm();
+
+		FormsAdminNavigator.openPublishURL(publishFormURL = "${publishedFormPublicURL}");
+
+
+		FormFields.editText(
+			fieldName = "Text",
+			fieldValue = "Test Text");
+
+		FormPortlet.submitLocalizedSuccessfully(buttonName = "Submit for Publication");
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsEntries.testcase:177]: Hey @PauloVAS, the last step planned for this test is `Assert the Success Page`, but this line is just asserting that a success message is being displayed. In light of this, can you please assert that the Success Page is also being displayed?

The macro `viewSuccessPage` will do it for you. :slightly_smiling_face:

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsEntries.testcase [hunk 2]
```diff
@@ -143,13 +143,54 @@ definition {
...
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoViewEntries();
+
+		FormsAdmin.viewFormTableBody(fieldBody = "Pending");
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsEntries.testcase:187]: @PauloVAS, can you please rename:

- `viewFormTableBody` to `viewFormEntryWorkflowStatus`
- `FORM_ENTRY_TABLE_BODY` to `FORM_ENTRY_WORKFLOW_STATUS`
- `key_fieldBody` to `key_formEntryWorkflowStatus`
- `fieldBody` to `formEntryWorkflowStatus`

This renaming will make your test use the same terms used on the Forms UI and code.

Thanks in advance. :slightly_smiling_face:

**Review by natocesarrego — CHANGES_REQUESTED** [review]

**Comment by natocesarrego** [comment]
@PauloVAS, can you please rebase your code against the latest master to solve the reported conflicts?

Thanks.

**Comment by PauloVAS** [comment]
@natocesarrego, fixed!

**Comment by natocesarrego** [comment]
Great job, @PauloVAS! :smile: :clap:

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**